environments/puppet/environment.conf

```diff
@@ -0,0 +1,17 @@
+# environment.conf - specify this environment's settings
```

Comment:
Should this be environment.conf.template, because you need to do a variable substitution to create a usable environment.conf?   Is it obvious to the user they need to do a substitution?